examples/readers/pytorch_data_api_tiledb_dense.ipynb

```diff
@@ -313,34 +313,35 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Train Epoch: 1 Batch: 0 Loss: 14.259003\n",
+      "Train Epoch: 1 Batch: 0 Loss: 7.341803\n",
       "Train Epoch: 1 Batch: 100 Loss: 2.302585\n",
       "Train Epoch: 1 Batch: 200 Loss: 2.302585\n",
       "Train Epoch: 1 Batch: 300 Loss: 2.302585\n",
       "Train Epoch: 1 Batch: 400 Loss: 2.302585\n",
-      "Train Epoch: 1 Batch: 500 Loss: 2.311349\n",
-      "Train Epoch: 1 Batch: 600 Loss: 1.965141\n",
-      "Train Epoch: 1 Batch: 700 Loss: 2.071231\n",
-      "Train Epoch: 1 Batch: 800 Loss: 1.871267\n",
-      "Train Epoch: 1 Batch: 900 Loss: 1.905761\n",
-      "Train Epoch: 2 Batch: 0 Loss: 1.951554\n",
-      "Train Epoch: 2 Batch: 100 Loss: 1.847873\n",
-      "Train Epoch: 2 Batch: 200 Loss: 1.860846\n",
-      "Train Epoch: 2 Batch: 300 Loss: 2.069581\n",
-      "Train Epoch: 2 Batch: 400 Loss: 1.885491\n",
-      "Train Epoch: 2 Batch: 500 Loss: 2.049599\n",
-      "Train Epoch: 2 Batch: 600 Loss: 2.052642\n",
-      "Train Epoch: 2 Batch: 700 Loss: 2.121343\n",
-      "Train Epoch: 2 Batch: 800 Loss: 2.021918\n",
-      "Train Epoch: 2 Batch: 900 Loss: 2.026648\n"
+      "Train Epoch: 1 Batch: 500 Loss: 2.302585\n",
+      "Train Epoch: 1 Batch: 600 Loss: 2.302585\n",
+      "Train Epoch: 1 Batch: 700 Loss: 2.302585\n",
+      "Train Epoch: 1 Batch: 800 Loss: 2.302585\n",
+      "Train Epoch: 1 Batch: 900 Loss: 2.302585\n",
+      "Train Epoch: 2 Batch: 0 Loss: 2.302585\n",
+      "Train Epoch: 2 Batch: 100 Loss: 2.302585\n",
+      "Train Epoch: 2 Batch: 200 Loss: 2.302585\n",
+      "Train Epoch: 2 Batch: 300 Loss: 2.302585\n",
+      "Train Epoch: 2 Batch: 400 Loss: 2.302585\n",
+      "Train Epoch: 2 Batch: 500 Loss: 2.302585\n",
+      "Train Epoch: 2 Batch: 600 Loss: 2.302585\n",
+      "Train Epoch: 2 Batch: 700 Loss: 2.302585\n",
+      "Train Epoch: 2 Batch: 800 Loss: 2.302585\n",
+      "Train Epoch: 2 Batch: 900 Loss: 2.302585\n"
      ]
     }
    ],
    "source": [
     "import torch.nn as nn\n",
     "import torch.optim as optim\n",
     "\n",
-    "from tiledb.ml.readers.pytorch import PyTorchTileDBDataLoader, ArrayParams\n",
+    "from tiledb.ml.readers.pytorch import PyTorchTileDBDataLoader\n",
+    "from tiledb.ml.readers.types import ArrayParams\n",
```

Comment:
These are leftovers from a previous branch. Can you rebase to the latest master?